src/Aspire.Hosting.Kubernetes/KubernetesPublisherOutputType.cs

```diff
@@ -0,0 +1,22 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace Aspire.Hosting.Kubernetes;
+
+/// <summary>
+/// Represents an output type for Kubernetes, such as Helm or Kustomize.
+/// This class is used to define the format or approach for publishing
+/// Kubernetes configurations.
+/// </summary>
+internal static class KubernetesPublisherOutputType
+{
+    public const string Helm = "helm";
+    public const string Kustomize = "kustomize";
+
+    public static string InvalidOutputTypeMessage(string outputType) =>
+        $"""
+        The '--output-type [type]' option was invalid. Supplied value was: '{outputType}'.
+        "Valid values are '{Helm}' and '{Kustomize}'.
```

Comment:
Are these sufficiently different that they should be different publishers?

Reply:
I don't think so. Flow will be identical, up to writing the output.
All options will be the same for instance

Reply:
Although that would probably change if we introduce modelled resources akin to what Azure.Provisioning produces, As we'd probably have the concept of things like HelmValue, HelmDictionaryOfT etc (and the same but Kustomize specific)